doc/sphinx/proofs/automatic-tactics/auto.rst

```diff
@@ -134,6 +134,12 @@ Programmable proof search
       Behaves like :tacn:`eauto` but shows the tactics it tries to solve the goal,
       including failing paths.
 
+   .. tacn:: dfs eauto {? @nat_or_var } {? @auto_using } {? @hintbases }
+
+      .. deprecated:: 8.15
```

Comment:
It's not urgent to remove, we already had to negotiate for deprecating `bfs eauto` in 8.15.0, so I think we can just target 8.16 for this one.